plugin_gems.rb

```diff
@@ -19,3 +19,4 @@
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
 download "fluent-plugin-google-cloud", "0.5.2"
+download "fluent-plugin-detect-exceptions", "0.0.1"
```

Comment:
Not yet. But you are right, ideally the new Gem should go in first and then I need to bump this version number here before submitting this change.